### PR TITLE
Set port to 8053 in BIND configuration example, fix typos

### PR DESCRIPTION
**Pull request by elnappo**

### CODE REVIEW

**Comment by cagedmantis** [comment]
Thank you for this submission @elnappo

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ bind_exporter next to BIND, so it's only necessary to open a port locally.
 
 ```
 statistics-channels {
-  inet 127.0.0.1 port 8080 allow { 127.0.0.1; };
+  inet 127.0.0.1 port 8053 allow { 127.0.0.1; };
```
Comment (cagedmantis) [inline README.md]: Port 8080 seems like the default bind statistics port. Do you have a compelling reason for this change?

**Comment by elnappo** [comment]
I've changed it based on https://github.com/digitalocean/bind_exporter/blob/master/bind_exporter.go#L438 and https://github.com/digitalocean/bind_exporter/issues/24#issuecomment-290298767

I'm not sure if there is something like a "default" port. BINDs documentation mentions port 8888 (https://ftp.isc.org/isc/bind9/cur/9.10/doc/arm/Bv9ARM.ch06.html#statistics), some how-tos/tools are using 8080 (https://kb.isc.org/article/AA-00769/0/Using-BINDs-XML-statistics-channels.html) or 8053 (https://collectd.org/wiki/index.php/Plugin:BIND).

It does not matter for me which port is the "default" port, but it should be consistent between `README.md` and the actual default value for the port flag.

**Comment by cagedmantis** [comment]
lgtm

**Pull request merged**